src/install/pre_install.sh

```diff
@@ -17,39 +17,45 @@ fi
 echo "Install Pre-Install Requirements"
 sudo apt-get -y install python3-pip git libffi-dev
 
-echo "Installing Docker"
-
-# Uninstall old versions
-sudo apt-get -y remove docker docker-engine docker.io
-
 # Install packages to allow apt to use a repository over HTTPS
 sudo apt-get -y install apt-transport-https ca-certificates curl software-properties-common
 
-if [ "${CODENAME}" = "stretch" ] || [ "${CODENAME}" = "buster" ]
-then
-    # Add Docker’s official GPG key
-    curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
+echo "Installing Docker"
 
-    # set up the stable repository
-    if [ ! -f /etc/apt/sources.list.d/docker.list ]
-    then
-        echo "deb [arch=amd64] https://download.docker.com/linux/debian ${CODENAME} stable" > docker.list
-        sudo mv docker.list /etc/apt/sources.list.d/docker.list
-    fi
+if [ "${CODENAME}" = "focal" ]
+then
+	sudo apt-get -y install docker docker-compose docker.io
```

Comment:
"docker.io" should be enough -- the ubuntu package "docker" seems to be some gnome GUI stuff https://packages.ubuntu.com/focal/docker